src/ImageBuilder/examples/Stop-AzImageBuilderTemplate.md

```diff
@@ -1,28 +1,26 @@
 ### Example 1: Stop image template creation
 ```powershell
-PS C:\> Start-AzImageBuilderTemplate -ImageTemplateName template-name-sn78hg -ResourceGroupName wyunchi-imagebuilder -AsJob 
+Start-AzImageBuilderTemplate -ImageTemplateName template-name-sn78hg -ResourceGroupName wyunchi-imagebuilder -AsJob 
 
 Id     Name            PSJobTypeName   State         HasMoreData     Location             Command
```

Comment:
move this output to Start-AzImageBuilderTemplate.md Example 1, then remove this output